build.sbt

```diff
@@ -423,10 +432,14 @@ lazy val dynamic = projectMatrix
   .jvmPlatform(
     allJvmScalaVersions,
     jvmDimSettings ++ Seq(
-      libraryDependencies += Dependencies.Smithy.model % Test
+      Test / fork := true,
+      // Forwarding cwd to tests
```

Comment:
the comment is wrong in the sense that this is now the cwd

Reply:
I'm not sure I follow. Care to elaborate ?